app/views/helpers/mainContentHeader.scala.html

```diff
@@ -0,0 +1,28 @@
+@*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@(backButtonEnabled: Boolean)(implicit messages: Messages)
+
+@backButton = {
+    @if(backButtonEnabled) {
+        <div style="width:40%; display: inline-block" align="left" class="form-group-compound">
+            <a id="back" class="back-link" href="javascript:history.back()">@messages("app.common.back")</a>
```

Comment:
This shouldn't require the wrapper div with the inline styles and updating the link class from .back-link to .link-back should update this to the latest style. But happy for this to wait for me to get around to updating styles for VAT pages to avoid any confusion or broken layouts etc.

Reply:
I think I copied that from VAT Frontend, there's a story about UI in the sprint. I guess you can fix that for both services once you'll be working on the story